test/WebSites/InlineConstraintsWebSite/Startup.cs

```diff
@@ -12,6 +13,26 @@ public class Startup
         // Set up application services
         public void ConfigureServices(IServiceCollection services)
         {
+            services.ConfigureRouting(
+                routeOptions => routeOptions.ConstraintMap.Add(
+                    "IsbnDigitScheme10",
+                    typeof(IsbnDigitScheme10Constraint)));
+
+            services.ConfigureRouting(
+                routeOptions => routeOptions.ConstraintMap.Add(
+                    "IsbnDigitScheme13",
+                    typeof(IsbnDigitScheme10Constraint)));
+
+            services.ConfigureRouting(
```

Comment:
@Eilon I'll add a comments like this is for testing the ConstraintMap for update. I talked to @rynowak  before and that was the one of the intentions of the bug.